data/mods/Magiclysm/monsters/feral_wizards.json

```diff
@@ -772,5 +772,219 @@
+    "dodge": 1,
+    "harvest": "human",
+    "dissect": "dissect_human_sample_single",
+    "vision_day": 45,
+    "vision_night": 3,
+    "stomach_size": 700,
+    "path_settings": { "max_dist": 45, "allow_open_doors": true, "avoid_traps": true, "avoid_sharp": true },
+    "special_attacks": [
+      {
+        "type": "monster_attack",
+        "attack_type": "melee",
+        "id": "feral_stormshaper_spell_lightning_blast",
+        "cooldown": 8,
+        "move_cost": 100,
+        "range": 9,
+        "accuracy": 4,
+        "damage_max_instance": [ { "damage_type": "electric", "amount": 20 } ],
+        "dodgeable": true,
+        "blockable": false,
+        "condition": { "not": { "u_has_flag": "NO_SPELLCASTING" } },
+        "hit_dmg_u": "%1$s gestures and sparks impacts your %2$s!",
+        "hit_dmg_npc": "%1$s gestures and sparks impact <npcname>'s %2$s!",
+        "miss_msg_u": "%1$s gestures and you narrowly avoid a shower of sparks!",
+        "miss_msg_npc": "%1$s gestures and <npcname> narrowly avoids a shower of sparks!",
+        "no_dmg_msg_u": "%1$s gestures your %2$s is impacted by a shower of sparks bolt, but they do no damage.",
+        "no_dmg_msg_npc": "%1$s gestures and <npcname>'s %2$s is impacted by a shower of sparks bolt, but they do no damage."
+      },
+      {
+        "id": "feral_stormshaper_wall_of_fog",
+        "type": "spell",
+        "spell_data": { "id": "stormshaper_wall_of_fog", "min_level": 5 },
+        "cooldown": 20,
+        "condition": { "not": { "u_has_flag": "NO_SPELLCASTING" } },
+        "monster_message": "%1$s gestures and fog rapidly condenses out of the air!"
+      },
+      {
+        "id": "feral_stormshaper_shocking dash",
+        "type": "spell",
+        "spell_data": { "id": "shocking_dash", "min_level": 5 },
+        "cooldown": 25,
+        "condition": { "not": { "u_has_flag": "NO_SPELLCASTING" } },
+        "monster_message": "%1$s surges forward in a burst of sparks."
+      },
+      {
+        "id": "feral_stormshaper_blinding_flash",
+        "type": "spell",
+        "spell_data": { "id": "blinding_flash", "min_level": 3 },
+        "cooldown": 15,
+        "condition": { "not": { "u_has_flag": "NO_SPELLCASTING" } },
+        "monster_message": "%1$s makes a quick gesture and disappears."
+      },
+      {
+        "id": "feral_stormshaper_summon_stormhammer",
+        "type": "spell",
+        "spell_data": { "id": "feral_stormshaper_summon_stormhammer_spell" },
+        "cooldown": 50,
+        "condition": { "and": [ { "not": { "u_has_effect": "effect_feral_stormshaper_stormshammer" } }, { "not": { "u_has_flag": "NO_SPELLCASTING" } } ] },
+        "monster_message": "%1$s snaps an arm out and a hammer made of lightning crackles into existence."
+      },
+      {
+        "type": "monster_attack",
+        "attack_type": "melee",
+        "id": "feral_stormshaper_spell_lightning_blast",
```

Comment:
I think the ID on this needs to change to something like "feral_stormshaper_stormhammer_attack" or something like that? I think you already used "feral_stormshaper_spell_lightning_blast" above.